client/src/components/UserForm.js

```diff
@@ -131,6 +131,8 @@ export default function UserForm() {
             <option value="">Select...</option>
             <option value="male">Male</option>
             <option value="Female">Female</option>
+            <option value="Transgender'">Transgender</option>
```

Comment:
**<option value="Transgender**'**">** in this line there is one single quote extra which sends the wrong data to the database, 
Please change this.
"Transgender"